src/components/InterruptedVideosGallery.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { RouteComponentProps } from '@reach/router'
+
+import { useQuery } from '@apollo/client'
+import { usePersonalData } from '@/hooks'
+import { GET_VIDEOS_WITH_IDS } from '@/api/queries'
+
+import { GetVideosWithIds, GetVideosWithIdsVariables } from '@/api/queries/__generated__/GetVideosWithIds'
+
+import { ErrorFallback, VideoGallery } from '@/components'
+
+interface Lookup {
+  [key: string]: number
+}
+
+const InterruptedVideosGallery: React.FC<RouteComponentProps> = () => {
+  const {
+    state: { watchedVideos },
+  } = usePersonalData()
+  const interruptedVideosState = watchedVideos.filter((video) => video.__typename === 'INTERRUPTED' && video.timestamp)
```

Comment:
That condition isn't needed anymore since we provide a default now

```suggestion
  const interruptedVideosState = watchedVideos.filter((video) => video.__typename === 'INTERRUPTED')
```